README.md

```diff
@@ -79,6 +79,56 @@ Blocks events for a duration after an event is successfully emitted.
 
 Like `Iterable.whereType` for a stream.
 
+### Comparison to Rx Operators
+
+The semantics and naming in this package have some overlap, and some conflict,
+with the [ReactiveX](https://reactivex.io/) suite of libraries. Some of the
+conflict is intentional - Dart `Stream` predates `Observable` and coherence with
+the Dart ecosystem semantics and naming is a strictly higher priority than
+consistency with ReactiveX.
+
+Rx Operator Category      | variation                                              | `stream_transform`
+------------------------- | ------------------------------------------------------ | ------------------
+[`sample`][rx_sample]     | `sample/throttleLast(Duration)`                        | No equivalent
+&#x200B;                  | `throttleFirst(Duration)`                              | [`throttle`][throttle]
+&#x200B;                  | `sample(Observable)`                                   | No equivalent
```

Comment:
`sample(Stream<void> trigger)` would be a fine thing to add. We'd need to think about either matching `buffer` or if we do `#145` then we'd probably want `sample` to have the same default `sample(Stream<void> trigger, {bool longPoll: true}`., and then reactive would be equivalent to `sample(trigger, longPoll: false)`

Then reactive's `sample/throttleLast(Duration)` could be `sample(Stream.periodic(duration), longPoll: false)`.